PKGBUILD.MSYS2

```diff
@@ -0,0 +1,64 @@
+# Themis build script for pacman in MSYS2 environment
+#
+# Maintainer: Cossack Labs Limited <dev@cossacklabs.com>
+
+pkgname=('themis' 'themis-devel')
+pkgbase=themis
+pkgver=0.11.2
+pkgrel=1
+
+pkgdesc="Data security library for network communication and data storage"
```

Comment:
"Cross-platform cryptographic library for mobile, desktop, cloud, and IoT; secures the data during authentication, messaging, APIs, storage, network exchange, etc."

Reply:
PKGBUILD docs [recommend](https://wiki.archlinux.org/index.php/PKGBUILD#pkgdesc) this description to be concise and generally fit into one line. I have reused the short description of our Debian and RPM packages:

- https://github.com/cossacklabs/themis/blob/5ea9e317f7bd4e6a94a6d69a4705bdd9bea7b53c/Makefile#L574

Reply:
```suggestion
pkgdesc="Cross-platform cryptographic library for protecting data during authentication, messaging, storage, network exchange, etc."
```

Reply:
but i'm ok if we merge your option